src/command-parser/expand-wildcard.ts

```diff
@@ -0,0 +1,109 @@
+import fs from 'fs';
+import _ from 'lodash';
+
+import { CommandInfo } from '../command';
+import { CommandParser } from './command-parser';
+
+const OMISSION = /\(!([^)]+)\)/;
```

Comment:
Would you mind adding a comment explaining what this Regexp does?
It isn't clear its intent unless you read the rest of the code and, even then, it's a bit confused.

Thanks in advance.

Reply:
Yeah, I was confused when I read it at first as well. Good catch, just added a comment. Do you think that clarifies it enough?

Reply:
Thank you.

I was just wondering why we'd need to look for `)` in the script command?!

I guess most of the people (myself included) don't know we could use something like this `deno task lint:*(!fix)` as a valid command.

Perhaps, adding an example in the comment would help as well.

Reply:
This behaviour is [currently already documented](https://github.com/open-cli-tools/concurrently/blob/main/docs/cli/shortcuts.md?plain=1#L61), so I feel this is a bit out of scope of this pull request?